docker/utils/utils.py

```diff
@@ -337,6 +337,33 @@ def convert_volume_binds(binds):
     return result
 
 
+def convert_tmpfs_mounts(tmpfs):
+    if isinstance(tmpfs, dict):
+        return tmpfs
+
+    if not isinstance(tmpfs, list):
+        raise ValueError(
+            'Tmpfs spec must be a list'
+        )
```

Comment:
There seems to be some inconsistencies in the error messages here. This one says must be a list, but the previous branch accepted dicts.  The next one says they must be something else.